appveyor.yml

```diff
@@ -18,4 +18,4 @@ test_script:
 - stack setup > nul
 # The ugly echo "" hack is to avoid complaints about 0 being an invalid file
 # descriptor
-- echo "" | stack --no-terminal build --haddock --no-haddock-deps --bench --test :validity-test --no-run-tests
+- echo "" | stack --no-terminal build --haddock --no-haddock-deps --bench --test :validity-test
```

Comment:
> Removed --no-run-tests and it fails now.

`:validity-test` is still broken on Windows (#74). As you're working on `:test` here, you'll want to test `:test`. :)